drake/examples/Pendulum/CMakeLists.txt

```diff
@@ -12,14 +12,23 @@ if (LCM_FOUND)
   add_test(NAME runPendulumDynamics COMMAND runPendulumDynamics WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/examples/Pendulum)
 
   add_executable(runPendulumEnergyShaping runEnergyShaping.cpp)
-  target_link_libraries(runPendulumEnergyShaping drakeSystem)
+  target_link_libraries(runPendulumEnergyShaping drakeSystem drakeGeometryUtil drakeRBM)
```

Comment:
drakeGeometryUtil should not be necessary since drakeRBM already links against that.